### PR TITLE
Add data provenance to system card

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Added data provenance to System, Model and Assessment Card to capture metadata in case these cards are generation from other source files.

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilizes.

## Developer Certificate of Origin

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ChristopherSpelt** [comment]
At this moment, there is information about the source data file, so we do not have 'real' data provenance because this should include all the following: (where is the data from, who made the change, when was it changed, who created this data). Any suggestions how we can add this? A problem is for example that a system card resulting from the algoritmeregister is generated from a file containing all entries from the algoritmeregister.

File: docs/projects/tad/reporting-standard/latest.md [hunk 1]
```diff
@@ -62,10 +62,21 @@ the examples in the next section.
...
-3. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+2. `provenance` (OPTIONAL). In case this System Card is generated from another source file, this field
+can capture the historical context of the contents of this System Card.
+
+    1. `modifier` (OPTIONAL, string). Git-hash of the file containing the transformations that where used
+    to fill in this System Card.
+    2. `timestamp` (OPTIONAL, string). A timestamp of the date, time and timezone of generation of this System Card.
+    Timestamp should be given, preferably in UTC (represented as `Z`), in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html)
+    format, i.e. `2024-04-16T16:48:14Z`.
+    3. `author` (OPTIONAL, string). Name or identifier of the person or system who initiated the
```
Comment (anneschuth) [inline docs/projects/tad/reporting-standard/latest.md:73]: I'm not sure we should conflate persons with systems. Could we have to separate fields maybe? 
"author" for a person and "repo" (or "tool" or "repo-uri" or just "uri") for a reference to the tool?

**Review by berrydenhartog — APPROVED** [review]
LGTM

File: docs/projects/tad/reporting-standard/latest.md (hunk 1 above)
Comment (ChristopherSpelt) [reply docs/projects/tad/reporting-standard/latest.md]: Added your suggestion.

File: docs/projects/tad/reporting-standard/latest.md [hunk 2]
```diff
@@ -62,10 +62,21 @@ the examples in the next section.
 A `system_card` contains the following information.
 
 1. `schema_version` (REQUIRED, string). Version of the schema used, for example "0.1a2".
-2. `name` (OPTIONAL, string). Name used to describe the system.
-3. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+2. `provenance` (OPTIONAL). In case this System Card is generated from another source file, this field
+can capture the historical context of the contents of this System Card.
+
+    1. `modifier` (OPTIONAL, string). Git-hash of the file containing the transformations that where used
```
Comment (anneschuth) [inline docs/projects/tad/reporting-standard/latest.md:68]: Reading this more closely, I don't think we mean a "git-hash of the file", but rather a "git commit hash"? And if that's the only type of value we support, shouldn't we just call it that?

**Review by anneschuth — APPROVED** [review]
Looks good, apart from small comment I left

File: docs/projects/tad/reporting-standard/latest.md (hunk 2 above)
Comment (ChristopherSpelt) [reply docs/projects/tad/reporting-standard/latest.md]: Added your suggested change

**Review by anneschuth — APPROVED** [review]

**Pull request merged**